packages/terra-tabs/src/Tabs.jsx

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
-import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
-import ContentContainer from 'terra-content-container';
-import ResponsiveElement from 'terra-responsive-element';
+import WorkspaceItem from './common-tabs/WorkspaceItem';
```

Comment:
can we rename this files and variable names to tabitem or commontabitem etc ?

Reply:
Changed in https://github.com/cerner/terra-framework/pull/1678/commits/af8e24e54807108ab41da3c81bd4e17f1f6f3d8d